### PR TITLE
[plugin-web-app] Do not prepend domains without subdomain separation with dot

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/ICookieManager.java [hunk 1]
```diff
@@ -26,8 +25,6 @@
 {
     void addCookie(String cookieName, String cookieValue, String path, String urlAsString);
 
-    void addCookie(String cookieName, String cookieValue, String path, URI uri);
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/ICookieManager.java:29]: this should go in a separate PR

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/ICookieManager.java (hunk 1 above)
Comment (uarlouski) [reply vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/ICookieManager.java:29]: https://github.com/vividus-framework/vividus/commit/35ae64da1162bf5200592ee20610e2a013d8965f

**Review by ikalinin1 — APPROVED** [review]

**Review by valfirst — APPROVED** [review]
